cmake/Subdirectories.cmake

```diff
@@ -207,7 +207,7 @@ function(resolve_gem_dependencies object_type object_path)
         set(user_external_subdir_option -ed "${user_external_subdirs}")
     endif()
     execute_process(COMMAND 
-        ${LY_PYTHON_CMD} "${LY_ROOT_FOLDER}/scripts/o3de/o3de/cmake.py" --${object_type_lower}-path "${object_path}" ${user_external_subdir_option}
+        ${LY_PYTHON_CMD} "${LY_ROOT_FOLDER}/scripts/o3de/o3de/cmake.py" --${object_type_lower}-path "${object_path}" --engine-path "${LY_ROOT_FOLDER}"
```

Comment:
This file is missing changes from this commit by @AMZN-alexpete https://github.com/o3de/o3de/pull/15835/commits/96b4292164cedf9371973ea5a0de5a5bdd2e7b1d

Reply:
good catch!